(tf_gpu36) davidr@login-gpu1:~/projects/saraGAN/SURFGAN$ mpirun -n 2 -npernode 2 python -u main.py pgan /nfs/managed_datasets/LIDC-IDRI/npy/average/ '(1, 128, 512, 512)' --starting_phase 3 --ending_phase 3 --base_dim 1024 --latent_dim 1024 --horovod --starting_alpha 0 --scratch_path /scratch --gpu
WARNING:tensorflow:From /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From main.py:565: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

WARNING:tensorflow:From main.py:565: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

WARNING:tensorflow:From main.py:568: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From main.py:568: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From main.py:587: The name tf.random.set_random_seed is deprecated. Please use tf.compat.v1.random.set_random_seed instead.

WARNING:tensorflow:From main.py:587: The name tf.random.set_random_seed is deprecated. Please use tf.compat.v1.random.set_random_seed instead.

Rank 0:0 reporting!
Rank 1:1 reporting!
WARNING:tensorflow:From main.py:35: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

WARNING:tensorflow:From main.py:52: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.

Arguments passed:
Namespace(activation='leaky_relu', architecture='pgan', base_batch_size=128, base_dim=1024, beta1=0, beta2=0.9, calc_metrics=False, continue_path=None, d_annealing=1, d_scaling='none', dataset_path='/nfs/managed_datasets/LIDC-IDRI/npy/average/', ema_beta=0.99, ending_phase=3, final_shape='(1, 128, 512, 512)', g_annealing=1, g_scaling='none', gp_weight=1, gpu=True, horovod=True, latent_dim=1024, leakiness=0.2, learning_rate=0.001, loss_fn='logistic', max_global_batch_size=128, mixing_nimg=262144, num_metric_samples=512, scratch_path='/scratch', seed=42, stabilizing_nimg=262144, starting_alpha=0.0, starting_phase=3)
Saving files to /home/davidr/projects/saraGAN/SURFGAN/runs/pgan/2020-01-21_08:54:49
WARNING:tensorflow:From main.py:52: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.

Length of dataset: 994
Length of dataset: 994
2020-01-21 09:55:05.340525: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
2020-01-21 09:55:05.344370: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
2020-01-21 09:55:05.502093: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:3b:00.0
2020-01-21 09:55:05.502239: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:3b:00.0
2020-01-21 09:55:05.505091: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 1 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:5e:00.0
2020-01-21 09:55:05.505159: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 1 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:5e:00.0
2020-01-21 09:55:05.508049: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 2 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:b1:00.0
2020-01-21 09:55:05.508119: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 2 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:b1:00.0
2020-01-21 09:55:05.511033: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 3 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:d9:00.0
2020-01-21 09:55:05.511104: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:05.511103: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 3 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:d9:00.0
2020-01-21 09:55:05.511165: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:05.541771: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-21 09:55:05.541767: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-21 09:55:05.582231: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-21 09:55:05.582588: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-21 09:55:05.632770: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-21 09:55:05.633645: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-21 09:55:05.691574: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-21 09:55:05.691577: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-21 09:55:05.729294: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-21 09:55:05.730739: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-21 09:55:05.780532: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-21 09:55:05.793310: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0, 1, 2, 3
2020-01-21 09:55:05.793738: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-01-21 09:55:05.797566: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-21 09:55:05.810359: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0, 1, 2, 3
2020-01-21 09:55:05.810767: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-01-21 09:55:08.071375: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4e98310 executing computations on platform CUDA. Devices:
2020-01-21 09:55:08.071419: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.071430: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (1): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.071439: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (2): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.071447: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (3): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.080509: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1700000000 Hz
2020-01-21 09:55:08.080648: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x54c7370 executing computations on platform Host. Devices:
2020-01-21 09:55:08.080664: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-01-21 09:55:08.097493: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:3b:00.0
2020-01-21 09:55:08.098776: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 1 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:5e:00.0
2020-01-21 09:55:08.099986: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 2 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:b1:00.0
2020-01-21 09:55:08.101212: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 3 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:d9:00.0
2020-01-21 09:55:08.101266: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:08.101315: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-21 09:55:08.101340: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-21 09:55:08.101363: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-21 09:55:08.101386: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-21 09:55:08.101408: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-21 09:55:08.101432: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-21 09:55:08.110783: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0, 1, 2, 3
2020-01-21 09:55:08.110839: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:08.218080: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5bbe2c0 executing computations on platform CUDA. Devices:
2020-01-21 09:55:08.218121: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.218131: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (1): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.218138: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (2): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.218145: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (3): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-01-21 09:55:08.221890: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1700000000 Hz
2020-01-21 09:55:08.222012: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x61fccc0 executing computations on platform Host. Devices:
2020-01-21 09:55:08.222025: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-01-21 09:55:08.224207: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:3b:00.0
2020-01-21 09:55:08.225405: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 1 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:5e:00.0
2020-01-21 09:55:08.226609: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 2 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:b1:00.0
2020-01-21 09:55:08.227848: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 3 with properties:
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.6705
pciBusID: 0000:d9:00.0
2020-01-21 09:55:08.227897: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:08.227965: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-21 09:55:08.227992: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-21 09:55:08.228010: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-21 09:55:08.228029: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-21 09:55:08.228046: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-21 09:55:08.228064: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-21 09:55:08.237602: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0, 1, 2, 3
2020-01-21 09:55:08.237661: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-21 09:55:08.915472: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-21 09:55:08.915534: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 1 2 3
2020-01-21 09:55:08.915546: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N Y Y Y
2020-01-21 09:55:08.915554: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 1:   Y N Y Y
2020-01-21 09:55:08.915561: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 2:   Y Y N Y
2020-01-21 09:55:08.915568: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 3:   Y Y Y N
2020-01-21 09:55:08.923865: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10334 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:3b:00.0, compute capability: 6.1)
2020-01-21 09:55:08.925884: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 10334 MB memory) -> physical GPU (device: 1, name: GeForce GTX 1080 Ti, pci bus id: 0000:5e:00.0, compute capability: 6.1)
2020-01-21 09:55:08.927890: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 10334 MB memory) -> physical GPU (device: 2, name: GeForce GTX 1080 Ti, pci bus id: 0000:b1:00.0, compute capability: 6.1)
2020-01-21 09:55:08.929885: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:3 with 10344 MB memory) -> physical GPU (device: 3, name: GeForce GTX 1080 Ti, pci bus id: 0000:d9:00.0, compute capability: 6.1)
WARNING:tensorflow:From main.py:84: DatasetV1.make_one_shot_iterator (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `for ... in dataset:` to iterate over a dataset. If using `tf.estimator`, return the `Dataset` object directly from your input function. As a last resort, you can use `tf.compat.v1.data.make_one_shot_iterator(dataset)`.
2020-01-21 09:55:09.080820: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-21 09:55:09.080875: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 1 2 3
2020-01-21 09:55:09.080887: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N Y Y Y
2020-01-21 09:55:09.080895: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 1:   Y N Y Y
2020-01-21 09:55:09.080902: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 2:   Y Y N Y
2020-01-21 09:55:09.080909: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 3:   Y Y Y N
2020-01-21 09:55:09.093877: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10332 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:3b:00.0, compute capability: 6.1)
2020-01-21 09:55:09.095894: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 10332 MB memory) -> physical GPU (device: 1, name: GeForce GTX 1080 Ti, pci bus id: 0000:5e:00.0, compute capability: 6.1)
2020-01-21 09:55:09.097905: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 10332 MB memory) -> physical GPU (device: 2, name: GeForce GTX 1080 Ti, pci bus id: 0000:b1:00.0, compute capability: 6.1)
2020-01-21 09:55:09.100401: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:3 with 10332 MB memory) -> physical GPU (device: 3, name: GeForce GTX 1080 Ti, pci bus id: 0000:d9:00.0, compute capability: 6.1)
WARNING:tensorflow:From main.py:84: DatasetV1.make_one_shot_iterator (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `for ... in dataset:` to iterate over a dataset. If using `tf.estimator`, return the `Dataset` object directly from your input function. As a last resort, you can use `tf.compat.v1.data.make_one_shot_iterator(dataset)`.
WARNING:tensorflow:From /home/davidr/projects/saraGAN/SURFGAN/networks/ops.py:87: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
Generator parameters: 26232833
Discriminator parameters:: 26219521
WARNING:tensorflow:From main.py:176: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

WARNING:tensorflow:From /home/davidr/projects/saraGAN/SURFGAN/networks/ops.py:87: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From main.py:176: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

[login-gpu1:174385] *** Process received signal ***
[login-gpu1:174385] Signal: Segmentation fault (11)
[login-gpu1:174385] Signal code: Address not mapped (1)
[login-gpu1:174385] Failing at address: 0x30
[login-gpu1:174384] *** Process received signal ***
[login-gpu1:174384] Signal: Segmentation fault (11)
[login-gpu1:174384] Signal code: Address not mapped (1)
[login-gpu1:174384] Failing at address: 0x30
[login-gpu1:174385] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x110e0)[0x2ad6c8e270e0]
[login-gpu1:174385] [login-gpu1:174384] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x110e0)[0x2b58846c50e0]
[login-gpu1:174384] [ 1] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/mpi_lib.cpython-36m-x86_64-linux-gnu.so(+0x948b0)[0x2b58e67118b0]
[ 1] [login-gpu1:174384] [ 2] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/mpi_lib.cpython-36m-x86_64-linux-gnu.so(+0x95eec)[0x2b58e6712eec]
[login-gpu1:174384] [ 3] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/mpi_lib.cpython-36m-x86_64-linux-gnu.so(+0x948b0)[0x2ad72ae738b0]
[login-gpu1:174385] [ 2] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/horovod/tensorflow/mpi_lib.cpython-36m-x86_64-linux-gnu.so(+0x95eec)[0x2ad72ae74eec]
[login-gpu1:174385] [ 3] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so.1(_ZN10tensorflow15shape_inference16InferenceContext3RunERKSt8functionIFNS_6StatusEPS1_EE+0x4d)[0x2b58d972da9d]
[login-gpu1:174384] [ 4] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow12ShapeRefiner10RunShapeFnEPKNS_4NodeEPKNS_18OpRegistrationDataEPNS_24ExtendedInferenceContextE+0x230)[0x2b58a78883b0]
[login-gpu1:174384] [ 5] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow12ShapeRefiner7AddNodeEPKNS_4NodeE+0xcb8)[0x2b58a7889ed8]
[login-gpu1:174384] [ 6] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(TF_FinishOperation+0x44a)[0x2b58a46d77ba]
[login-gpu1:174384] [ 7] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(+0x29917d6)[0x2b58a1f4a7d6]
[login-gpu1:174384] [ 8] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyCFunction_FastCallDict+0x16b)[0x2b5883f3391b]
[login-gpu1:174384] [ 9] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149880)[0x2b5883fa8880]
[login-gpu1:174384] [10] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1ff5)[0x2b5883faaef5]
[login-gpu1:174384] [11] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x14887f)[0x2b5883fa787f]
[login-gpu1:174384] [12] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149765)[0x2b5883fa8765]
[login-gpu1:174384] [13] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1ff5)[0x2b5883faaef5]
[login-gpu1:174384] [14] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149354)[0x2b5883fa8354]
[login-gpu1:174384] [15] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyFunction_FastCallDict+0x12a)[0x2b5883fb0e2a]
[login-gpu1:174384] [16] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallDict+0x14e)[0x2b5883eec03e]
[login-gpu1:174384] [17] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_Call_Prepend+0xcd)[0x2b5883eec24d]
[login-gpu1:174384] [18] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyObject_Call+0x48)[0x2b5883eebe08]
[login-gpu1:174384] [19] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xeb3ec)[0x2b5883f4a3ec]
[login-gpu1:174384] [20] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xe7ac2)[0x2b5883f46ac2]
[login-gpu1:174384] [21] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallDict+0x89)[0x2b5883eebf79]
[login-gpu1:174384] [22] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallKeywords+0x62)[0x2b5883eec4c2]
[login-gpu1:174384] [23] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x14968d)[0x2b5883fa868d]
[login-gpu1:174384] [24] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x2075)[0x2b5883faaf75]
[login-gpu1:174384] [25] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149354)[0x2b5883fa8354]
[login-gpu1:174384] [26] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyEval_EvalCodeEx+0x3e)[0x2b5883fa890e]
[login-gpu1:174384] [27] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xb5f75)[0x2b5883f14f75]
[login-gpu1:174384] [28] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyObject_Call+0x48)[0x2b5883eebe08]
[login-gpu1:174384] [29] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1c14)[0x2b5883faab14]
[login-gpu1:174384] *** End of error message ***
/home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so.1(_ZN10tensorflow15shape_inference16InferenceContext3RunERKSt8functionIFNS_6StatusEPS1_EE+0x4d)[0x2ad71de8fa9d]
[login-gpu1:174385] [ 4] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow12ShapeRefiner10RunShapeFnEPKNS_4NodeEPKNS_18OpRegistrationDataEPNS_24ExtendedInferenceContextE+0x230)[0x2ad6ebfea3b0]
[login-gpu1:174385] [ 5] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow12ShapeRefiner7AddNodeEPKNS_4NodeE+0xcb8)[0x2ad6ebfebed8]
[login-gpu1:174385] [ 6] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(TF_FinishOperation+0x44a)[0x2ad6e8e397ba]
[login-gpu1:174385] [ 7] /home/davidr/envs/tf_gpu36/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so(+0x29917d6)[0x2ad6e66ac7d6]
[login-gpu1:174385] [ 8] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyCFunction_FastCallDict+0x16b)[0x2ad6c869591b]
[login-gpu1:174385] [ 9] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149880)[0x2ad6c870a880]
[login-gpu1:174385] [10] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1ff5)[0x2ad6c870cef5]
[login-gpu1:174385] [11] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x14887f)[0x2ad6c870987f]
[login-gpu1:174385] [12] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149765)[0x2ad6c870a765]
[login-gpu1:174385] [13] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1ff5)[0x2ad6c870cef5]
[login-gpu1:174385] [14] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149354)[0x2ad6c870a354]
[login-gpu1:174385] [15] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyFunction_FastCallDict+0x12a)[0x2ad6c8712e2a]
[login-gpu1:174385] [16] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallDict+0x14e)[0x2ad6c864e03e]
[login-gpu1:174385] [17] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_Call_Prepend+0xcd)[0x2ad6c864e24d]
[login-gpu1:174385] [18] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyObject_Call+0x48)[0x2ad6c864de08]
[login-gpu1:174385] [19] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xeb3ec)[0x2ad6c86ac3ec]
[login-gpu1:174385] [20] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xe7ac2)[0x2ad6c86a8ac2]
[login-gpu1:174385] [21] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallDict+0x89)[0x2ad6c864df79]
[login-gpu1:174385] [22] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyObject_FastCallKeywords+0x62)[0x2ad6c864e4c2]
[login-gpu1:174385] [23] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x14968d)[0x2ad6c870a68d]
[login-gpu1:174385] [24] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x2075)[0x2ad6c870cf75]
[login-gpu1:174385] [25] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0x149354)[0x2ad6c870a354]
[login-gpu1:174385] [26] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyEval_EvalCodeEx+0x3e)[0x2ad6c870a90e]
[login-gpu1:174385] [27] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(+0xb5f75)[0x2ad6c8676f75]
[login-gpu1:174385] [28] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(PyObject_Call+0x48)[0x2ad6c864de08]
[login-gpu1:174385] [29] /sw/arch/Debian9/EB_production/2019/software/Python/3.6.6-foss-2018b/lib/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x1c14)[0x2ad6c870cb14]
[login-gpu1:174385] *** End of error message ***
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 0 with PID 0 on node login-gpu1 exited on signal 11 (Segmentation fault).
--------------------------------------------------------------------------
